src/jit/importer.cpp

```diff
@@ -2928,12 +2928,12 @@ CORINFO_CLASS_HANDLE Compiler::impGetObjectClass()
 /* static */
 void Compiler::impBashVarAddrsToI(GenTree* tree1, GenTree* tree2)
 {
-    if (tree1->IsVarAddr())
+    if (tree1->IsLocalAddrExpr() != nullptr)
```

Comment:
Why do you compare `bool IsLocalAddrExpr` with `nullptr`?

Reply:
It's not `bool`:
https://github.com/dotnet/coreclr/blob/master/src/jit/gentree.h#L1854